src/components/CardContainer/CardContainer.js

```diff
@@ -16,7 +20,7 @@ class CardContainer extends React.Component {
     }
 
     getDogs = () => {
-        fetch('http://localhost:3000/dogs', {
+        fetch(fetchUrl, {
```

Comment:
At the moment if we try and fetch and get a 500 we don't let the user know - we need some sort of error message so that they know and don't keep refreshing the page